gulpfile.js

```diff
@@ -29,6 +29,30 @@ gulp.task('build', function() {
       .pipe(gulp.dest('dist'));
 });
 
+gulp.task('clean', function() {
+  return del(['dist']);
+});
+
+function serve() {
+  var app = require('express')();
+  var webserver = require('gulp-webserver');
```

Comment:
the dependency doesnt exist.

run `yarn add --dev gulp-webserver` and `yarn add -dev express`